packaging/curated-cloud-installer.spec

```diff
@@ -0,0 +1,137 @@
+#
+# spec file for package curated-cloud-installer
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+%define         app_dir      %{_datadir}/curated-cloud-installer
+%define         lib_dir      %{_libdir}/curated-cloud-installer
+%define         data_dir     %{_sharedstatedir}/curated-cloud-installer
+
+%define         ruby_version ruby3.0
+%define         k8s_version  kubernetes1.18
+
+Name:           curated-cloud-installer
+Version:        0.0.1
+Release:        0
+Summary:        Curated Cloud Installation of Rancher on AWS EKS
+License:        GPL-3.0
+Group:          Productivity/Networking/Web/Frontends
+URL:            http://www.github.com/suse-enceladus/curated-cloud-installer
+Source0:        %{name}-%{version}.tar.bz2
```

Comment:
This should point to the URL of the upstream tarball once that is available. If you want the tarball to be stored using a certain filename, you can use this template `%{url}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz`.